apps/web/src/components/launch-darkly/LaunchDarklyProvider.tsx

```diff
@@ -2,33 +2,23 @@ import * as Sentry from '@sentry/react';
 
 import { IOrganizationEntity } from '@novu/shared';
 import { asyncWithLDProvider } from 'launchdarkly-react-client-sdk';
-import { PropsWithChildren, ReactNode, useEffect, useMemo, useRef, useState } from 'react';
+import { PropsWithChildren, useEffect, useMemo, useRef, useState } from 'react';
 import { useFeatureFlags, useAuthContext, LAUNCH_DARKLY_CLIENT_SIDE_ID } from '@novu/shared-web';
 import { selectShouldInitializeLaunchDarkly } from './utils/selectShouldInitializeLaunchDarkly';
-import { selectShouldShowLaunchDarklyFallback } from './utils/selectShouldShowLaunchDarklyFallback';
 
 /** A provider with children required */
 type GenericLDProvider = Awaited<ReturnType<typeof asyncWithLDProvider>>;
 
 /** Simply renders the children */
 const DEFAULT_GENERIC_PROVIDER: GenericLDProvider = ({ children }) => <>{children}</>;
 
-export interface ILaunchDarklyProviderProps {
-  /** Renders when LaunchDarkly is enabled and is awaiting initialization */
-  fallbackDisplay: ReactNode;
-}
-
```

Comment:
Not needed, this is moved to the `AuthProvider`. This is suitable because Launch Darkly was actually dependent on auth context being available, now it will always be there.